Mac OS Catalina OTP/29-rc1 linking error

Linking **trace_ip_drv.so** error with **OTP 29.0rc1** and default Xcode's Clang: 11.0.3 build 1103

trace_ip_drv.so
**Undefined symbols for architecture x86_64**:
  "_erl_nif_user_trace_i4s4", referenced from:
      _user_trace_i4s4 in dyntrace.o
  "_erl_nif_user_trace_n", referenced from:
      _user_trace_n in dyntrace.o
  "_erl_nif_user_trace_s1", referenced from:
      _user_trace_s1 in dyntrace.o
**ld: symbol(s) not found for architecture x86_64**


```bash
./configure
--disable-debug
--disable-dependency-tracking
--prefix=/usr/local/Cellar/erlang/29.0rc1
--libdir=/usr/local/Cellar/erlang/29.0rc1/lib
--enable-dynamic-ssl-lib
--with-odbc=/usr/local/opt/unixodbc
--with-ssl=/usr/local/opt/openssl@3
--without-javac
--with-wx-config=/usr/local/opt/wxwidgets@3.2/bin/wx-config-3.2
--enable-darwin-64bit
--enable-kernel-poll
--with-dynamic-trace=dtrace
```

======================
2026-02-12 16:36:56 +0000

make
```make
 MAKE	depend
 MAKE	generate
 DTRACE	x86_64-apple-darwin19.6.0/erlang_dtrace.h
 GEN	x86_64-apple-darwin19.6.0/opt/emu/OPCODES-GENERATED
 GEN	x86_64-apple-darwin19.6.0/opt/emu/TABLES-GENERATED
 GEN	x86_64-apple-darwin19.6.0/opt/emu/erl_alloc_types.h
 GEN	x86_64-apple-darwin19.6.0/erl_version.h
 GEN	x86_64-apple-darwin19.6.0/opt/emu/driver_tab.c
 GEN	x86_64-apple-darwin19.6.0/opt/emu/preload.c
 MAKE	opt
 GEN	obj/x86_64-apple-darwin19.6.0/opt/depend.mk
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/lib_src/yielding_c_fun/x86_64-apple-darwin19.6.0/ycf_lexer.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/lib_src/yielding_c_fun/x86_64-apple-darwin19.6.0/ycf_main.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/lib_src/yielding_c_fun/x86_64-apple-darwin19.6.0/ycf_node.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/lib_src/yielding_c_fun/x86_64-apple-darwin19.6.0/ycf_parser.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/lib_src/yielding_c_fun/x86_64-apple-darwin19.6.0/ycf_printers.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/lib_src/yielding_c_fun/x86_64-apple-darwin19.6.0/ycf_string.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/lib_src/yielding_c_fun/x86_64-apple-darwin19.6.0/ycf_symbol.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/lib_src/yielding_c_fun/x86_64-apple-darwin19.6.0/ycf_utils.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/lib_src/yielding_c_fun/x86_64-apple-darwin19.6.0/ycf_yield_fun.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/r/ethr_aux.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/r/ethr_atomics.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/r/ethr_mutex.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/r/ethr_cbf.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/r/ethread.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/r/ethr_event.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/erl_printf_format.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/erl_printf.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/erl_misc_utils.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/r/erl_printf_format.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/r/erl_printf.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/r/erl_misc_utils.o
 LD	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/lib_src/yielding_c_fun/bin/x86_64-apple-darwin19.6.0/yielding_c_fun
 AR	../lib/internal/x86_64-apple-darwin19.6.0/libethread.a
 AR	../lib/internal/x86_64-apple-darwin19.6.0/liberts_internal.a
 AR	../lib/internal/x86_64-apple-darwin19.6.0/liberts_internal_r.a
 RANLIB	../lib/internal/x86_64-apple-darwin19.6.0/liberts_internal.a
 RANLIB	../lib/internal/x86_64-apple-darwin19.6.0/liberts_internal_r.a
 RANLIB	../lib/internal/x86_64-apple-darwin19.6.0/libethread.a
 GEN	obj/x86_64-apple-darwin19.6.0/opt/MADE
 GEN	x86_64-apple-darwin19.6.0/opt/emu/erl_map.ycf.h
 GEN	x86_64-apple-darwin19.6.0/opt/emu/erl_db_insert_list.ycf.h
 GEN	x86_64-apple-darwin19.6.0/opt/emu/utils.ycf.h
 GEN	x86_64-apple-darwin19.6.0/opt/emu/GENERATED
 MAKE	depend
 GEN	x86_64-apple-darwin19.6.0/opt/emu/jit.depend.mk
 GEN	x86_64-apple-darwin19.6.0/opt/emu/src.depend.mk
 GEN	x86_64-apple-darwin19.6.0/opt/emu/drv.depend.mk
 GEN	x86_64-apple-darwin19.6.0/opt/emu/nif.depend.mk
 GEN	x86_64-apple-darwin19.6.0/opt/emu/sys.depend.mk
 GEN	x86_64-apple-darwin19.6.0/opt/emu/target.depend.mk
 GEN	x86_64-apple-darwin19.6.0/opt/emu/zlib.depend.mk
 GEN	x86_64-apple-darwin19.6.0/opt/emu/ryu.depend.mk
 GEN	x86_64-apple-darwin19.6.0/opt/emu/zstd.depend.mk
 GEN	x86_64-apple-darwin19.6.0/opt/emu/depend.mk
 MAKE	depend
make[4]: Nothing to be done for `depend'.
make[2]: Nothing to be done for `depend'.
 MAKE	depend
make[2]: Nothing to be done for `depend'.
 MAKE	erl_interface_build
=== Entering application erl_interface
 MAKE	opt
 GEN	x86_64-apple-darwin19.6.0/depend.mk
 CC	x86_64-apple-darwin19.6.0/depend.mk
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/ei_connect.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/ei_resolve.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/eirecv.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/send.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/send_exit.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/send_reg.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/decode_atom.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/decode_big.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/decode_bignum.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/decode_binary.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/decode_boolean.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/decode_char.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/decode_double.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/decode_fun.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/decode_intlist.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/decode_iodata.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/decode_list_header.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/decode_long.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/decode_pid.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/decode_port.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/decode_ref.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/decode_skip.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/decode_string.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/decode_trace.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/decode_tuple_header.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/decode_ulong.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/decode_version.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/decode_longlong.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/decode_ulonglong.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/encode_atom.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/encode_big.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/encode_bignum.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/encode_binary.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/encode_boolean.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/encode_char.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/encode_double.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/encode_fun.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/encode_list_header.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/encode_long.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/encode_pid.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/encode_port.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/encode_ref.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/encode_string.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/encode_trace.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/encode_tuple_header.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/encode_ulong.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/encode_version.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/encode_longlong.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/encode_ulonglong.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/epmd_port.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/epmd_publish.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/epmd_unpublish.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/ei_decode_term.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/ei_format.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/ei_locking.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/ei_malloc.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/ei_portio.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/ei_printterm.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/ei_pthreads.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/ei_trace.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/ei_x_encode.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/get_type.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/show_msg.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/ei_compat.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/ei_init.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/ei_cmp_nc.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/global_names.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/global_register.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/global_unregister.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/global_whereis.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.st/x86_64-apple-darwin19.6.0/md5_dgst.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/ei_connect.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/ei_resolve.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/eirecv.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/send.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/send_exit.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/send_reg.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/decode_atom.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/decode_big.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/decode_bignum.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/decode_binary.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/decode_boolean.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/decode_char.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/decode_double.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/decode_fun.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/decode_intlist.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/decode_iodata.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/decode_list_header.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/decode_long.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/decode_pid.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/decode_port.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/decode_ref.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/decode_skip.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/decode_string.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/decode_trace.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/decode_tuple_header.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/decode_ulong.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/decode_version.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/decode_longlong.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/decode_ulonglong.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/encode_atom.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/encode_big.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/encode_bignum.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/encode_binary.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/encode_boolean.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/encode_char.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/encode_double.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/encode_fun.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/encode_list_header.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/encode_long.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/encode_pid.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/encode_port.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/encode_ref.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/encode_string.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/encode_trace.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/encode_tuple_header.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/encode_ulong.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/encode_version.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/encode_longlong.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/encode_ulonglong.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/epmd_port.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/epmd_publish.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/epmd_unpublish.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/ei_decode_term.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/ei_format.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/ei_locking.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/ei_malloc.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/ei_portio.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/ei_printterm.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/ei_pthreads.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/ei_trace.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/ei_x_encode.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/get_type.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/show_msg.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/ei_compat.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/ei_init.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/ei_cmp_nc.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/global_names.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/global_register.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/global_unregister.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/global_whereis.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj.mt/x86_64-apple-darwin19.6.0/md5_dgst.o
 AR	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei_st.a
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei_st.a(decode_bignum.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei_st.a(decode_long.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei_st.a(decode_ulong.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei_st.a(encode_bignum.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei_st.a(encode_long.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei_st.a(encode_ulong.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei_st.a(ei_locking.o) has no symbols
 RANLIB	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei_st.a
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei_st.a(decode_bignum.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei_st.a(decode_long.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei_st.a(decode_ulong.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei_st.a(encode_bignum.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei_st.a(encode_long.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei_st.a(encode_ulong.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei_st.a(ei_locking.o) has no symbols
 AR	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei.a
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei.a(decode_bignum.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei.a(decode_long.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei.a(decode_ulong.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei.a(encode_bignum.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei.a(encode_long.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei.a(encode_ulong.o) has no symbols
 RANLIB	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei.a
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei.a(decode_bignum.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei.a(decode_long.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei.a(decode_ulong.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei.a(encode_bignum.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei.a(encode_long.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/obj/x86_64-apple-darwin19.6.0/libei.a(encode_ulong.o) has no symbols
 LD	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/lib/erl_interface/bin/x86_64-apple-darwin19.6.0/erl_call
make[2]: Nothing to be done for `opt'.
=== Leaving application erl_interface
 MAKE	emulator
 MAKE	opt
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_main.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/preload.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_process.o
 EMU_CC	obj/x86_64-apple-darwin19.6.0/opt/emu/beam_emu.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/emu_load.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/beam_common.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/beam_bif_load.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/beam_bp.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/beam_catches.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/beam_debug.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/beam_load.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/beam_opcodes.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/beam_ranges.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/beam_transform_helpers.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/code_ix.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_alloc.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_alloc_util.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_goodfit_alloc.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_bestfit_alloc.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_afit_alloc.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_init.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_atom_table.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_bif_table.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_bif_ddll.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_bif_guard.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_bif_info.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_bif_op.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_bif_os.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_bif_lists.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_bif_persistent.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_bif_atomics.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_bif_counters.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_bif_trace.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_bif_unique.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_guard_bifs.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_dirty_bif_wrap.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_trace.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/copy.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/utils.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/bif.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/io.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_printf_term.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_debug.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_debugger.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_message.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_proc_sig_queue.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_process_dict.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_process_lock.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_port_task.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_arith.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/time.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_time_sup.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/external.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/dist.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/binary.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_db.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_db_util.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_db_hash.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_db_tree.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_thr_progress.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/big.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/hash.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/index.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/atom.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/module.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/export.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/register.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/break.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_async.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_lock_check.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_dyn_lock_check.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_gc.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_lock_count.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_posix_str.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_bits.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_math.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_fun.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_bif_port.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_term.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_node_tables.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_monitor_link.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_process_dump.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_hl_timer.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_cpu_topology.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_drv_thread.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_bif_chksum.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_bif_re.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_unicode.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/packet_parser.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/safe_hash.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_zlib.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_nif.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_bif_binary.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_ao_firstfit_alloc.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_thr_queue.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_sched_spec_pre_alloc.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_ptab.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_map.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_msacc.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_lock_flags.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_io_queue.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_db_catree.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/socket_dbg.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/socket_tarray.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/socket_util.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_flxctr.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_nfunc_sched.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_global_literals.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/beam_file.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/beam_types.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_term_hashing.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_bif_coverage.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_iolist.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_etp.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_internal_test.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_record.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/sys.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/sys_drivers.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/sys_env.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/sys_uds.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/driver_tab.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/unix_prim_file.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/sys_float.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/sys_time.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/sys_signal_stack.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/unix_socket_syncio.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_poll.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_check_io.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_mseg.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_mmap.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_osenv.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_unix_sys_ddll.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_sys_common_misc.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_os_monotonic_time_extender.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erlang_lttng.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/inet_drv.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/ram_file_drv.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/prim_tty_nif.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_tracer_nif.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/prim_buffer_nif.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/prim_file_nif.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/zlib_nif.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/zstd_nif.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/prim_socket_nif.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/prim_net_nif.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/obj/x86_64-apple-darwin19.6.0/opt/pcre2_auto_possess.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/obj/x86_64-apple-darwin19.6.0/opt/pcre2_chartables.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/obj/x86_64-apple-darwin19.6.0/opt/pcre2_chkdint.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/obj/x86_64-apple-darwin19.6.0/opt/pcre2_compile.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/obj/x86_64-apple-darwin19.6.0/opt/pcre2_compile_cgroup.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/obj/x86_64-apple-darwin19.6.0/opt/pcre2_compile_class.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/obj/x86_64-apple-darwin19.6.0/opt/pcre2_config.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/obj/x86_64-apple-darwin19.6.0/opt/pcre2_context.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/obj/x86_64-apple-darwin19.6.0/opt/pcre2_error.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/obj/x86_64-apple-darwin19.6.0/opt/pcre2_extuni.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/obj/x86_64-apple-darwin19.6.0/opt/pcre2_find_bracket.o
 GEN	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/pcre2_match_yield_coverage.gen.h
 GEN	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/pcre2_match_loop_break_cases.gen.h
 GEN	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/pcre2_match_memchr_break_cases.gen.h
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/obj/x86_64-apple-darwin19.6.0/opt/pcre2_match_data.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/obj/x86_64-apple-darwin19.6.0/opt/pcre2_newline.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/obj/x86_64-apple-darwin19.6.0/opt/pcre2_ord2utf.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/obj/x86_64-apple-darwin19.6.0/opt/pcre2_pattern_info.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/obj/x86_64-apple-darwin19.6.0/opt/pcre2_script_run.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/obj/x86_64-apple-darwin19.6.0/opt/pcre2_serialize.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/obj/x86_64-apple-darwin19.6.0/opt/pcre2_string_utils.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/obj/x86_64-apple-darwin19.6.0/opt/pcre2_study.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/obj/x86_64-apple-darwin19.6.0/opt/pcre2_substring.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/obj/x86_64-apple-darwin19.6.0/opt/pcre2_tables.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/obj/x86_64-apple-darwin19.6.0/opt/pcre2_ucd.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/obj/x86_64-apple-darwin19.6.0/opt/pcre2_valid_utf.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/obj/x86_64-apple-darwin19.6.0/opt/pcre2_xclass.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/ryu/obj/x86_64-apple-darwin19.6.0/opt/d2s.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/openssl/obj/x86_64-apple-darwin19.6.0/opt/md5_dgst.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/debug.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/entropy_common.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/error_private.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/fse_decompress.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/pool.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/threading.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/xxhash.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/zstd_common.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/fse_compress.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/hist.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/huf_compress.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/zstd_compress.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/zstd_compress_literals.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/zstd_compress_sequences.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/zstd_compress_superblock.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/zstd_double_fast.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/zstd_fast.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/zstd_lazy.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/zstd_ldm.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/zstd_opt.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/zstd_preSplit.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/zstdmt_compress.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/huf_decompress.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/zstd_ddict.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/zstd_decompress.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/zstd_decompress_block.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/huf_decompress_amd64.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_child_setup.o
 CC	obj/x86_64-apple-darwin19.6.0/opt/emu/erl_poll.flbk.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/obj/x86_64-apple-darwin19.6.0/opt/pcre2_match.o
 AR	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/ryu/obj/x86_64-apple-darwin19.6.0/opt/libryu.a
 AR	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/openssl/obj/x86_64-apple-darwin19.6.0/opt/micro-openssl.a
 LD	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/erl_child_setup
 AR	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/zstd/obj/x86_64-apple-darwin19.6.0/opt/libzstd.a
 AR	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/emulator/pcre/obj/x86_64-apple-darwin19.6.0/opt/libepcre.a
 LD	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/beam.emu
 AR	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/libbeam.a
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/libbeam.a(erl_lock_check.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/libbeam.a(erl_dyn_lock_check.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/libbeam.a(erl_lock_count.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/libbeam.a(safe_hash.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/libbeam.a(erlang_lttng.o) has no symbols
 RANLIB	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/libbeam.a
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/libbeam.a(erl_lock_check.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/libbeam.a(erl_dyn_lock_check.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/libbeam.a(erl_lock_count.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/libbeam.a(safe_hash.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/libbeam.a(erlang_lttng.o) has no symbols
/usr/bin/install -c /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/beam.emu /private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/beam.smp
 MAKE	opt
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/obj/x86_64-apple-darwin19.6.0/inet_gethost.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/obj/x86_64-apple-darwin19.6.0/heart.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/obj/x86_64-apple-darwin19.6.0/erlexec.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/obj/x86_64-apple-darwin19.6.0/typer.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/obj/x86_64-apple-darwin19.6.0/dialyzer.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/obj/x86_64-apple-darwin19.6.0/erlc.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/obj/x86_64-apple-darwin19.6.0/escript.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/obj/x86_64-apple-darwin19.6.0/ct_run.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/obj/x86_64-apple-darwin19.6.0/safe_string.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/obj/x86_64-apple-darwin19.6.0/run_erl.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/obj/x86_64-apple-darwin19.6.0/to_erl.o
 LD	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/erl_call
 LD	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/erl_call
 VSN	Install
 VSN	erl.src
 LD	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/inet_gethost
 LD	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/heart
 LD	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/erlexec
 LD	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/typer
 LD	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/dialyzer
 LD	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/erlc
 LD	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/escript
 LD	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/ct_run
 LD	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/to_erl
 LD	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/run_erl
 GEN	etp-commands
 MAKE	opt
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/obj/x86_64-apple-darwin19.6.0/epmd.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/obj/x86_64-apple-darwin19.6.0/epmd_srv.o
 CC	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/erts/obj/x86_64-apple-darwin19.6.0/epmd_cli.o
 LD	/private/tmp/erlang-20260212-2905-8jnwio/otp_src_29.0/bin/x86_64-apple-darwin19.6.0/epmd
 MAKE	opt
make[4]: Nothing to be done for `all'.
(cd preloaded/src && /Applications/Xcode.app/Contents/Developer/usr/bin/make ../../ebin/erts.app)
 GEN	deps/deps.mk
/bin/sh: erlc: command not found
make[3]: `../../ebin/erts.app' is up to date.
 MAKE	secondary_bootstrap
=== Entering application parsetools
make[3]: Nothing to be done for `opt'.
make[3]: Nothing to be done for `opt'.
=== Leaving application parsetools
=== Entering application sasl
make[3]: Nothing to be done for `opt'.
make[3]: Nothing to be done for `opt'.
make[3]: Nothing to be done for `opt'.
=== Leaving application sasl
make[2]: Nothing to be done for `opt'.
 MAKE	tertiary_bootstrap
=== Entering application parsetools
make[3]: Nothing to be done for `opt'.
make[3]: Nothing to be done for `opt'.
=== Leaving application parsetools
=== Entering application wx
 ERLC	../ebin/gl.beam
=== Leaving application wx
=== Entering application public_key
make[3]: Nothing to be done for `opt'.
make[3]: Nothing to be done for `opt'.
make[3]: Nothing to be done for `opt'.
make[3]: Nothing to be done for `opt'.
=== Leaving application public_key
=== Entering application erl_interface
 MAKE	opt
make[4]: Nothing to be done for `opt'.
make[3]: Nothing to be done for `opt'.
=== Leaving application erl_interface
=== Entering application syntax_tools
make[3]: Nothing to be done for `opt'.
make[3]: Nothing to be done for `opt'.
make[3]: Nothing to be done for `opt'.
=== Leaving application syntax_tools
=== Entering application snmp
make[3]: Nothing to be done for `opt'.
make[3]: Nothing to be done for `opt'.
make[4]: Nothing to be done for `opt'.
make[4]: Nothing to be done for `opt'.
make[4]: Nothing to be done for `opt'.
make[5]: Nothing to be done for `behaviour_targets'.
make[5]: Nothing to be done for `targets'.
make[5]: Nothing to be done for `behaviour_targets'.
make[5]: Nothing to be done for `targets'.
make[4]: Nothing to be done for `opt'.
make[4]: Nothing to be done for `opt'.
make[4]: Nothing to be done for `opt'.
make[4]: Nothing to be done for `opt'.
make[3]: Nothing to be done for `opt'.
=== Leaving application snmp
 MAKE	erts
 ERLC	prim_eval.abstr
 GEN	../../ebin/prim_eval.beam
 MAKE	libs
=== Entering application stdlib
make[3]: Nothing to be done for `opt'.
make[3]: Nothing to be done for `opt'.
make[3]: Nothing to be done for `opt'.
=== Leaving application stdlib
=== Entering application sasl
make[3]: Nothing to be done for `opt'.
make[3]: Nothing to be done for `opt'.
make[3]: Nothing to be done for `opt'.
=== Leaving application sasl
=== Entering application kernel
make[3]: Nothing to be done for `opt'.
make[3]: Nothing to be done for `opt'.
make[3]: Nothing to be done for `opt'.
=== Leaving application kernel
=== Entering application compiler
make[3]: Nothing to be done for `opt'.
make[3]: Nothing to be done for `opt'.
=== Leaving application compiler
=== Entering application tools
make[3]: Nothing to be done for `opt'.
make[3]: Nothing to be done for `opt'.
make[3]: Nothing to be done for `opt'.
make[3]: Nothing to be done for `opt'.
=== Leaving application tools
=== Entering application common_test
make[3]: Nothing to be done for `opt'.
make[3]: Nothing to be done for `opt'.
 MAKE	opt
make[4]: Nothing to be done for `debug'.
make[3]: Nothing to be done for `opt'.
=== Leaving application common_test
=== Entering application runtime_tools
make[3]: Nothing to be done for `opt'.
 MAKE	opt
 CC	../priv/obj/x86_64-apple-darwin19.6.0/dyntrace.o
 CC	../priv/obj/x86_64-apple-darwin19.6.0/trace_file_drv.o
 CC	../priv/obj/x86_64-apple-darwin19.6.0/trace_ip_drv.o
 LD	../priv/lib/x86_64-apple-darwin19.6.0/trace_file_drv.so
 LD	../priv/lib/x86_64-apple-darwin19.6.0/dyntrace.so
 LD	../priv/lib/x86_64-apple-darwin19.6.0/trace_ip_drv.so
Undefined symbols for architecture x86_64:
  "_erl_nif_user_trace_i4s4", referenced from:
      _user_trace_i4s4 in dyntrace.o
  "_erl_nif_user_trace_n", referenced from:
      _user_trace_n in dyntrace.o
  "_erl_nif_user_trace_s1", referenced from:
      _user_trace_s1 in dyntrace.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[4]: *** [../priv/lib/x86_64-apple-darwin19.6.0/dyntrace.so] Error 1
make[4]: *** Waiting for unfinished jobs....
rm ../priv/obj/x86_64-apple-darwin19.6.0/trace_file_drv.o ../priv/obj/x86_64-apple-darwin19.6.0/trace_ip_drv.o ../priv/obj/x86_64-apple-darwin19.6.0/dyntrace.o
make[3]: *** [opt] Error 2
make[2]: *** [opt] Error 2
make[1]: *** [opt] Error 2
make: *** [libs] Error 2
```

```bash
HOMEBREW_VERSION: 5.0.7
ORIGIN: https://github.com/Homebrew/brew
HEAD: b64029d4f999b8963d0dadb925d3b484207c1cee
Last commit: 8 weeks ago
Branch: HEAD
Core tap HEAD: 208a0cb11001c33da2cec42164d5dd1a738fc322
Core tap last commit: 7 weeks ago
Core tap branch: HEAD
Core tap JSON: 24 Dec 11:06 UTC
Core cask tap origin: https://github.com/Homebrew/homebrew-cask.git
Core cask tap HEAD: bb4487fa9a809e6b73a34d2e789029c59839cc54
Core cask tap last commit: 12 months ago
Core cask tap JSON: 30 Jan 16:04 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_DOWNLOAD_CONCURRENCY: 8
HOMEBREW_MAKE_JOBS: 4
HOMEBREW_NO_AUTO_UPDATE: set
HOMEBREW_NO_BOOTSNAP: set
HOMEBREW_NO_INSTALL_CLEANUP: set
HOMEBREW_NO_INSTALL_FROM_API: set
HOMEBREW_NO_INSTALL_UPGRADE: set
Homebrew Ruby: 3.4.8 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/3.4.8/bin/ruby
CPU: quad-core 64-bit haswell
Clang: 11.0.3 build 1103
Git: 2.51.0 => /usr/local/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 10.15.7-x86_64
CLT: 26.0
Xcode: 11.7

HOMEBREW_CC: clang
HOMEBREW_CXX: clang++
MAKEFLAGS: -j4
CMAKE_PREFIX_PATH: /usr/local/opt/m4:/usr/local
CMAKE_INCLUDE_PATH: /Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers
CMAKE_LIBRARY_PATH: /Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/System/Library/Frameworks/OpenGL.framework/Versions/Current/Libraries
PKG_CONFIG_PATH: /usr/local/opt/openssl@3/lib/pkgconfig:/usr/local/opt/unixodbc/lib/pkgconfig:/usr/local/opt/jpeg-turbo/lib/pkgconfig:/usr/local/opt/libpng/lib/pkgconfig:/usr/local/opt/xz/lib/pkgconfig:/usr/local/opt/lz4/lib/pkgconfig:/usr/local/opt/zstd/lib/pkgconfig:/usr/local/opt/libtiff/lib/pkgconfig:/usr/local/opt/pcre2/lib/pkgconfig
PKG_CONFIG_LIBDIR: /usr/lib/pkgconfig:/usr/local/Homebrew/Library/Homebrew/os/mac/pkgconfig/10.15
HOMEBREW_MAKE_JOBS: 4
HOMEBREW_GIT: git
HOMEBREW_SDKROOT: /Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk
ACLOCAL_PATH: /usr/local/share/aclocal
PATH: /usr/local/Homebrew/Library/Homebrew/shims/mac/super:/usr/local/opt/openssl@3/bin:/usr/local/opt/m4/bin:/usr/local/opt/libtool/bin:/usr/local/opt/unixodbc/bin:/usr/local/opt/jpeg-turbo/bin:/usr/local/opt/libpng/bin:/usr/local/opt/xz/bin:/usr/local/opt/lz4/bin:/usr/local/opt/zstd/bin:/usr/local/opt/libtiff/bin:/usr/local/opt/pcre2/bin:/usr/local/opt/wxwidgets@3.2/bin:/usr/bin:/bin:/usr/sbin:/sbin
```

--- a/.github/workflows/main.yaml
+++ b/.github/workflows/main.yaml
@@ -249,7 +249,7 @@ jobs:
           tar -xzf ./otp_src.tar.gz
           export PATH=$PWD/wxWidgets/release/bin:$PATH
           cd otp
-          $GITHUB_WORKSPACE/.github/scripts/build-macos.sh build_docs --disable-dynamic-ssl-lib
+          $GITHUB_WORKSPACE/.github/scripts/build-macos.sh build_docs --disable-dynamic-ssl-lib --with-dynamic-trace=dtrace
           tar -czf otp_macos_$(cat OTP_VERSION)_x86-64.tar.gz -C release .
 
       - name: Test Erlang

--- a/erts/emulator/beam/erl_nif.h
+++ b/erts/emulator/beam/erl_nif.h
@@ -378,7 +378,6 @@ extern void enif_ioq_destroy(ErlNifIOQueue *q);
 #    define ERL_NIF_API_FUNC_DECL(RET_TYPE, NAME, ARGS) ERL_NAPI_EXPORT extern RET_TYPE NAME ARGS
 #  endif
 #  include "erl_nif_api_funcs.h"
-#  undef ERL_NIF_API_FUNC_DECL
 #endif
 
 #if (defined(__WIN32__) || defined(_WIN32) || defined(_WIN32_))
@@ -447,10 +446,12 @@ ERL_NIF_INIT_EPILOGUE
 #endif
 
 #ifdef HAVE_USE_DTRACE
-ERL_NIF_TERM erl_nif_user_trace_s1(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[]);
-ERL_NIF_TERM erl_nif_user_trace_i4s4(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[]);
-ERL_NIF_TERM erl_nif_user_trace_n(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[]);
+ERL_NIF_API_FUNC_DECL(ERL_NIF_TERM,erl_nif_user_trace_s1,(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[]));
+ERL_NIF_API_FUNC_DECL(ERL_NIF_TERM,erl_nif_user_trace_i4s4,(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[]));
+ERL_NIF_API_FUNC_DECL(ERL_NIF_TERM,erl_nif_user_trace_n,(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[]));
 #endif
+
+#undef ERL_NIF_API_FUNC_DECL
 
 #endif /* __ERL_NIF_H__ */
 

